src/display/editor/tools.js

```diff
@@ -1488,6 +1506,23 @@ class AnnotationEditorUIManager {
     }
   }
 
+  showAllEditors(type, visible, updateButton = false) {
+    for (const editor of this.#allEditors.values()) {
+      if (editor.editorType === type) {
+        editor.show(visible);
+      }
+    }
+    if (
+      (this.#showAllStates?.get(
+        AnnotationEditorParamsType.HIGHLIGHT_SHOW_ALL
+      ) ?? true) !== visible
+    ) {
```

Comment:
This isn't all that readable at a glance, please use a temporary variable instead.
```suggestion
    const state =
      this.#showAllStates?.get(AnnotationEditorParamsType.HIGHLIGHT_SHOW_ALL) ??
      true;
    if (state !== visible) {   
```